walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/rawrequests/VerifiedIdOpenIdJwtRawRequest.kt

```diff
@@ -24,9 +27,31 @@ internal class VerifiedIdOpenIdJwtRawRequest(
             throw MissingRequestStateException("Request State is missing in presentation request")
         if (rawRequest.content.redirectUrl.isEmpty())
             throw MissingCallbackUrlException("Callback url is missing in presentation request")
+        val requirementsList = rawRequest.getPresentationDefinitions().map { it.toRequirement() }
+        val requirement = if (requirementsList.size == 1) {
+            requirementsList.first()
+        } else {
+            var groupRequirement = true
+            var groupOperator = GroupRequirementOperator.ALL
+            val requirements: List<Requirement> = requirementsList.map {
+                /// primary verifiable presentation
+                if (it is GroupRequirement) {
+                    groupRequirement = it.required
+                    groupOperator = it.requirementOperator
```

Comment:
If there are 2 group requirements, one in each VPToken, this would pick the groupOperator from the last requirement. When we flatten it, it gets complicated to assign the operator. Could we stick with just ALL for now?